client/my-sites/google-my-business/index.js

```diff
@@ -46,4 +46,15 @@ export default function() {
 		makeLayout,
 		clientRender
 	);
+
+	if ( config.isEnabled( 'google-my-business' ) ) {
+		page(
+			'/google-my-business/:site/select-location',
```

Comment:
Shouldn't `:site` always be the last segment? I think we have it wrong in the `/new` page too.

Reply:
That's what I thought as well. I'd encourage you to look at the discussion in https://github.com/Automattic/wp-calypso/pull/23730, and especially [this comment](https://github.com/Automattic/wp-calypso/pull/23730#issuecomment-377212122) which explains why we should have the site fragment here.

Reply:
After discussion in Slack, I think we should move `:site` to the last fragment.

Reply:
Alright, I'll take care of that in another pull request.

Reply:
👍